### PR TITLE
Remove Zeebe REST API

### PR DESCRIPTION
**Pull request by akeller**
## Description

🪦 Removing the deprecated Zeebe REST API. 🪦 

I think this can be separate from a specific alpha, so it can be reviewed and merged whenever (unless @tmetzke or @aleksander-dytko think differently).

Remaining items to bring this PR out of draft status:
* Redirects (to the Camunda REST API)
* Backporting (content removal only, not announcement)

@camunda/tech-writers one of you can take this over while I'm gone to get it merged.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] I (will) added a redirect for a renamed or deleted page to the .htaccess file.

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
The build currently breaks in part due to the migration guide for user tasks, work that I see happening over on https://github.com/camunda/camunda-docs/pull/4573/.

**Comment by tmetzke** [comment]
Do we also remove this from the `/api` directory? There are some references to that. Not sure if this needs to be part of this PR or is a follow-up since that is more about internal structure changes than user-facing docs.

**Comment by christinaausley** [comment]
> Do we also remove this from the `/api` directory? There are some references to that. Not sure if this needs to be part of this PR or is a follow-up since that is more about internal structure changes than user-facing docs.

I would assume so. Once I get https://github.com/camunda/camunda-docs/pull/4573 merged in I'll get this cleaned up and merged as well.

**Review by tmetzke — DISMISSED** [review]
Looks good to me, content-wise 👍 
I'm not sure if there are other structural leftovers for the Zeebe API, I guess @pepopowitz knows this best 🙂

File: static/.htaccess [hunk 1]
```diff
@@ -95,6 +95,9 @@ RewriteRule ^docs/reference/bpmn-processes/?(.*)$ /docs/components/modeler/bpmn/
 #   8.7: content moves introduced prior to the release of version 8.7.
 #---------------------------------------------------------------------------------
 
+# Remove Zeebe REST API
+RewriteRule ^docs/next/apis-tools/zeebe-api-rest/specifications/?$ /docs/next/apis-tools/camunda-api-rest/specifications/$1 [R=301,L]
+
```
Comment (pepopowitz) [inline static/.htaccess:100]: I think this needs to be more broad, no? The Zeebe REST API overview, auth, and tutorial pages are all siblings of `/specifications`.

**Review by pepopowitz — COMMENTED** [review]
In addition to the inline comment, I also _think_ [this block](https://github.com/camunda/camunda-docs/blob/d8732a60474ea62c52fac0918a8e831b7cf9e307/docs/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md?plain=1#L374-L379) is what's giving you a build error right now, and should be removed.

**Comment by christinaausley** [comment]
@tmetzke Can you confirm if the Zeebe REST API should be removed in 8.5 and 8.6?

**Comment by tmetzke** [comment]
> @tmetzke Can you confirm if the Zeebe REST API should be removed in 8.5 and 8.6?

As far as I know, we keep it there. @felix-mueller (covering for @aleksander-dytko), can you confirm or deny? 😄

**Comment by christinaausley** [comment]
I've confirmed with @felix-mueller this is still valid for 8.5 and 8.6, so this PR is ready for final review. 

@tmetzke Would you like me to keep this API in Postman for now, or remove it with an announcement to migrate to the Camunda 8 API?

**Comment by tmetzke** [comment]
> @tmetzke Would you like me to keep this API in Postman for now, or remove it with an announcement to migrate to the Camunda 8 API?

Let's add an announcement to move to C8 API 👍

File: versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md [hunk 2]
```diff
@@ -370,12 +370,6 @@ type:"link",
 href:"/docs/next/apis-tools/tasklist-api-rest/tasklist-api-rest-overview/",
 label: "Tasklist API (REST)",
 docId:"apis-tools/tasklist-api-rest/tasklist-api-rest-overview"
-},
-{
-type:"link",
-href:"/docs/next/apis-tools/zeebe-api-rest/zeebe-api-rest-overview/",
-label: "Zeebe API (REST)",
-docId:"apis-tools/zeebe-api-rest/zeebe-api-rest-overview"
```
Comment (tmetzke) [inline versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md:378]: @christinaausley, should we really just remove this from here? Should we rather adjust this to point to the C8 REST API?

**Review by tmetzke — DISMISSED** [review]
👍 Looks good to me, thanks for tracking it down everywhere 🚀 
🔧 Only one spot left that might need some attention.

Pre-approving since my last suggestion is not blocking.

File: versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md (hunk 2 above)
Comment (christinaausley) [reply versioned_docs/version-8.5/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md:378]: This versioned change snuck in. Will revert.

**Review by pepopowitz — APPROVED** [review]
Nice work!!!

image

**Pull request merged**